.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.x'
+      - '[0-9]+.[0-9]+.x'
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    timeout-minutes: 2
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        # additional validation steps can be run here
+      - name: Semantic Release
+        uses: BrightspaceUI/actions/semantic-release@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.D2L_GITHUB_TOKEN }}
```

Comment:
Assuming we still need to create this - @dlockhart can you help when you're back tomorrow?

Reply:
There's no branch protection rules set up for this repo, so technically it shouldn't be required. If no branch protection is what you'd like, this line will need to be switched to just `secrets.GITHUB_TOKEN` and the line above with `persist-credentials: false` can be removed. Otherwise let me know and I can add in the `D2L_GITHUB_TOKEN`.

Reply:
I think the protections are a good idea, but it's not really my repo.  Was just doing some dragon slaying in between other tasks.

Since this repo exists without the protections i'll use `secrets.GITHUB_TOKEN`